### PR TITLE
Update openssl-smime.pod.in

### PR DESCRIPTION
**Pull request by DominikNeubauer**
remove duplicate entries for -nocerts and -noattr

The manpage for _openssl-smime_ shows two entries for -nocerts and -noattr, respectively. One set of them (l. 198 ff) is quite exhaustive, the second one merely mentions the existence, and thus is redundant.

##### Checklist

- [X] documentation is added or updated
- [ ] tests are added or updated

CLA: trivial

### CODE REVIEW

File: doc/man1/openssl-smime.pod.in [hunk 1]
```diff
@@ -245,14 +245,6 @@ used multiple times if more than one signer is required. If a message is being
 verified then the signers certificates will be written to this file if the
 verification was successful.
 
-=item B<-nocerts>
-
-Don't include signers certificate when signing.
-
-=item B<-noattr>
-
-Don't include any signed attributes when signing.
```
Comment (slontis) [inline doc/man1/openssl-smime.pod.in:254]: Well spotted,
Maybe fix up the other ones so they are just as clear as these ones are.. There needs to at least be a comma in the -nocerts one above.

**Review by t8m — COMMENTED** [review]
Could you please also amend the commit message to include the `CLA: trivial` annotation?

**Comment by DominikNeubauer** [comment]
> Could you please also amend the commit message to include the `CLA: trivial` annotation?

Have I put it in on the wrong place?

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
> > Could you please also amend the commit message to include the `CLA: trivial` annotation?
> 
> Have I put it in on the wrong place?

Yes, it needs to be in the commit message, not just in the pull request description. You can use `git commit --amend ; git push --force`.

**Review by t8m — APPROVED** [review]
OK with CLA trivial

@paulidale Can you please confirm, you're OK with CLA trivial?

**Review by paulidale — APPROVED** [review]
okay with trivial

**Comment by t8m** [comment]
Merged to all the active branches. Thank you for your contribution.

**Pull request closed without merging**